src/components/pages/auth/verification.tsx

```diff
@@ -18,56 +17,45 @@ export function Verification() {
   const router = useRouter()
   const { flow: flowId, return_to: returnTo } = router.query
 
+  const emailVerifiedSuccessfully = strings.auth.messages[1080002]
```

Comment:
ah, that was what you meant! 😆
Sorry for overcomplicating it

Reply:
No worries 😀 Pretty interesting solution you found there with the `get`. 
Never saw that before.